pipelines/release/release.yaml

```diff
@@ -178,16 +142,10 @@ spec:
           - name: pathInRepo
             value: tasks/apply-mapping/apply-mapping.yaml
       params:
-        - name: extraConfigPath
-          value: "$(context.pipelineRun.uid)/extraConfig/$(params.extraConfigPath)"
         - name: failOnEmptyResult
           value: "true"
         - name: snapshotPath
```

Comment:
Can you add the RPA path?

Reply:
https://github.com/redhat-appstudio/release-service-catalog/blob/main/tasks/apply-mapping/apply-mapping.yaml#L19